SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLBStrategy.h

```diff
@@ -34,6 +34,8 @@ class SPATIALGDK_API UAbstractLBStrategy : public UObject
 
 	virtual void Init() {}
 
+	virtual void Init(UWorld* World) {}
```

Comment:
This interface is used by multiple classes and feels cleaner to overload than change the interface, especially if we only merge this to NWX.

Reply:
I will remove the Init(World*). I added it because I was getting a NULL pointer to the world from the GetWorld() inside the LayeredLBStrategy.
After investigating the problem further I was not creating the load balancing strategy correctly. Now I have fixed the route of the problem this overload is not required.